search/src/test/scala/weco/api/search/works/WorksFiltersTest.scala

```diff
@@ -580,202 +548,102 @@ class WorksFiltersTest
   }
 
   describe("filtering works by license") {
-    def createLicensedWork(
-      licenses: Seq[License]
-    ): Work.Visible[WorkState.Indexed] = {
-      val items =
-        licenses.map { license =>
-          createDigitalItemWith(license = Some(license))
-        }.toList
-
-      indexedWork().items(items)
-    }
-
-    val ccByWork = createLicensedWork(licenses = List(License.CCBY))
-    val ccByNcWork = createLicensedWork(licenses = List(License.CCBYNC))
-    val bothLicenseWork =
-      createLicensedWork(licenses = List(License.CCBY, License.CCBYNC))
-    val noLicenseWork = createLicensedWork(licenses = List.empty)
-
-    val works = List(ccByWork, ccByNcWork, bothLicenseWork, noLicenseWork)
-
     it("filters by license") {
       withWorksApi {
         case (worksIndex, routes) =>
-          insertIntoElasticsearch(worksIndex, works: _*)
-          assertJsonResponse(
-            routes,
-            s"$rootPath/works?items.locations.license=cc-by"
-          ) {
-            Status.OK -> worksListResponse(
-              works = Seq(ccByWork, bothLicenseWork).sortBy {
-                _.state.canonicalId
-              }
-            )
-          }
-      }
-    }
+          indexTestDocuments(worksIndex, worksLicensed: _*)
 
-    it("filters by multiple licenses") {
-      withWorksApi {
-        case (worksIndex, routes) =>
-          insertIntoElasticsearch(worksIndex, works: _*)
           assertJsonResponse(
             routes,
-            s"$rootPath/works?items.locations.license=cc-by,cc-by-nc"
+            path = s"$rootPath/works?items.locations.license=cc-by"
           ) {
-            Status.OK -> worksListResponse(
-              works = Seq(ccByWork, ccByNcWork, bothLicenseWork).sortBy {
-                _.state.canonicalId
-              }
-            )
-          }
-      }
-    }
-  }
-
-  describe("filtering works by items.locations.license") {
```

Comment:
It removed this block of tests because it was a literal duplicate of the license filtering tests above, but with a different `describe` name.